src/main/kotlin/com/csbroker/apiserver/repository/common/RedisRepository.kt

```diff
@@ -32,4 +34,25 @@ class RedisRepository(
     fun removePasswordVerification(code: String) {
         redisTemplate.delete(code)
     }
+
+    fun setRank(scoreMap: Map<UUID, Double>) {
+        scoreMap.forEach {
+            redisTemplate.opsForZSet().add("ranking", it.key.toString(), it.value)
+        }
+    }
+
+    fun getRank(userId: UUID): RankResultDto {
+        var rank = 0L
+
+        val score = redisTemplate.opsForZSet().score("ranking", userId.toString()) ?: 0.0
+        val rankKeys = redisTemplate.opsForZSet().reverseRangeByScore("ranking", score, score, 0, 1)
+
+        if (rankKeys != null) {
+            for (rankKey in rankKeys) {
+                rank = redisTemplate.opsForZSet().reverseRank("ranking", rankKey!!)!!
+            }
+        }
```

Comment:
```suggestion
        val rankKey = redisTemplate.opsForZSet().reverseRangeByScore("ranking", score, score, 0, 1).first()

        if (rankKey != null) {
            rank = redisTemplate.opsForZSet().reverseRank("ranking", rankKey!!)!!
        }
```
size가 하나 있는 List를 위해 for문을 쓰는 것 보다는 first를 써서 명확하게 코드 의도를 알려주는게 더 좋을 것 같아요!!
리뷰를 하면서 어떤 의도를 가진 코드일까 고민하는데 꽤 오랜 시간이 걸렸거든요 ㅠㅠ